Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/06-aks-supporting.md

```diff
@@ -1,12 +1,12 @@
 # Create resources that support AKS
 
 The following will be created:
-* [Azure Container Registry](./06-AKS-supporting/supporting-infra.tf)
+* [Azure Container Registry](ACR)(./06-AKS-supporting/supporting-infra.tf)
```

Comment:
This is a syntax error.  Consider using my recommendation at https://github.com/Azure/aks-baseline-windows/pull/46#discussion_r1068225986